var/spack/repos/builtin/packages/uriparser/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Uriparser(CMakePackage):
+    """uriparser is a strictly RFC 3986 compliant URI parsing and handling
+    library written in C89 ("ANSI C")."""
+
+    homepage = "https://uriparser.github.io/"
+    url      = "https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.3/uriparser-0.9.3.tar.gz"
+
+    version('0.9.3', sha256='6cef39d6eaf1a48504ee0264ce85f078758057dafb1edd0a898183b55ff76014')
+
+    variant('docs', default=False, description='Build API documentation')
+
+    depends_on('cmake@3.3:', type='build')
+    depends_on('googletest@1.8.1:', type='link')
```

Comment:
`googletest` is only needed at test time. However, my understanding of the `type='test'` dependency type is that it only adds `prefix.bin` to the `PATH`. In this case, `googletest` has no `prefix.bin`, only a `prefix.lib` and `prefix.include` that need to be linked to during testing.

I switched this to a `type='link'` deptype, but it still isn't being RPATHed correctly. Still trying to investigate why.